aspnetcore/web-api/index.md

```diff
@@ -453,7 +510,43 @@ To make automatic and custom responses consistent, call the <xref:Microsoft.AspN
 
 ### Log automatic 400 responses
 
-See [How to log automatic 400 responses on model validation errors (dotnet/AspNetCore.Docs#12157)](https://github.com/dotnet/AspNetCore.Docs/issues/12157).
+You can add logging by utilizing [InvalidModelStateResponseFactory](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.apibehavioroptions.invalidmodelstateresponsefactory). Here is an example of code to be used in `Startup.ConfigureServices` that adds logging on top of **any** already existing functionality that generates the response:
```

Comment:
Should we give a bit more details about the default framework implementation here (that it returns for example `SerializableError` or anything else note-worthy?) I have no idea if other ASP.NET Core versions after 2.2 and prior to 3.0 change this behavior.